src/geovista/examples/clouds.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+"""Importable and runnable geovista example.
+
+Notes
+-----
+.. versionadded:: 0.4.0
+
+"""
+from __future__ import annotations
+
+import cmocean
+from matplotlib.colors import LinearSegmentedColormap
+
+import geovista as gv
+from geovista.pantry import cloud_amount
+import geovista.theme  # noqa: F401
+
+#: The colormap to render the clouds.
+CMAP = cmocean.cm.gray
+
+#: Multiplication factor of the zlevel for cloud surface stratification.
```

Comment:
```suggestion
#: Multiplication factor of the z-level for cloud surface stratification.
```

Reply:
Up to this point I've always referenced `zlevel` as `zlevel` and never `z-level`.

If you prefer that, then we should be consistent and change it everywhere in the codebase.

Personally, I prefer keeping things simple and using `zlevel` everywhere ... what do you think?

Reply:
Oops. I forgot to resolve this. I'll zlevel it to match the rest.